pkg/apis/experimental/v1alpha1/custom_cluster_builder_validation.go

```diff
@@ -0,0 +1,26 @@
+package v1alpha1
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
+)
+
+func (ccb *CustomClusterBuilder) SetDefaults(context.Context) {
+}
+
+func (ccb *CustomClusterBuilder) Validate(ctx context.Context) *apis.FieldError {
+	return ccb.Spec.CustomBuilderSpec.Validate(ctx).
+		Also(validateServiceAccountRef(ccb.Spec.ServiceAccountRef))
```

Comment:
Super minor but, for consistency could we do?
`validateServiceAccountRef(ccb.Spec.ServiceAccountRef).ViaField("spec", "serviceAccountRef")`

Reply:
Or nest it and allow the CustomClusterBuilderSpec have a validation method?